backend/src/main/resources/org/cryptomator/hub/flyway/V7__Audit_Events.sql

```diff
@@ -0,0 +1,20 @@
+-- Cryptomator Hub 1.1.0
+
+CREATE TABLE "audit_event"
+(
+	"id"        BIGSERIAL,
```

Comment:
I would prefer the "long" sequence declaration (see mentioned link in https://github.com/cryptomator/hub/pull/192#discussion_r1190883179), so we can _explicitly_ choose a sequence name in a `CREATE SEQUENCE` statement, which will then be used in `AuditEvent.java`'s `@SequenceGenerator`.

Reply:
Fixed in 7f14e11217ec6396448ec46008f6a3ff9af6fd2d